### PR TITLE
Add batch delete for TaskRuns and PipelineRuns

### PR DESCRIPTION
**Pull request by AlanGreene**
# Changes

Resolves https://github.com/tektoncd/dashboard/issues/1752

Add support for batch delete of TaskRuns and PipelineRuns from
their respective list views.

When the user selects rows, the 'Delete' action will appear in
the table header, and when activated will display a confirmation
modal listing the selected runs.

Individual API requests will be made to delete each selected
run. This follows the same pattern already in place for both
Secrets and PipelineResources. We may look into batch delete
support at the API level in future if this causes problems.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: packages/components/src/components/PipelineRuns/PipelineRuns.js [hunk 1]
```diff
@@ -52,8 +52,7 @@ const PipelineRuns = ({
   },
   getPipelineRunCreatedTime = pipelineRun =>
     pipelineRun.metadata.creationTimestamp,
-  getPipelineRunId = pipelineRun =>
-    `${pipelineRun.metadata.namespace}:${pipelineRun.metadata.name}`,
+  getPipelineRunId = pipelineRun => pipelineRun.metadata.uid,
```
Comment (AlanGreene) [inline packages/components/src/components/PipelineRuns/PipelineRuns.js:55]: @steveodonovan I'm not sure why we were using namespace+name before, makes more sense to me to use the uid and not seeing any other impact of this change other than it being easier to resolve selected rows to actual resources. We're already using uid for TaskRuns.

**Review by steveodonovan — APPROVED** [review]
/approve
/lgtm

**Review by steveodonovan — APPROVED** [review]
/approve
/lgtm

**Pull request merged**